taylorseager/CulinaryChronicles
# View Recipe Details Finalized

## Description
The view recipe details page includes image, name, total time, servings, categories, ingredients, directions and if it is a favorite. It also has an edit button. 

## Related Issue
#45

## Motivation and Context
The user needs to be able to view recipe details and then be able to direct to the edit form if necessary.

## How Can This Be Tested?
View recipes page -> 
select edit on a recipe card -> 
should route to view recipe details page
edit button should route to edit form

## Types of changes
